javascript/code-challenges/arrayBinarySearch/array-binary-search.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+function BinarySearch(arr, key){
+  let start = 0;
+  let end = arr.length - 1;
+  
+  while (start <= end){
+    let mid = Math.floor((start + end) / 2);
+
+    if(key === arr[mid]){
+      return mid;
+    }
+    if(key < arr[mid]){
+      end = mid - 1;
+    }
+    if(key > arr[mid]){
+      start = mid + 1;
+    }
+  }
+  return -1;
+}
```

Comment:
at the end add `module.exports = binarySearch;` allows you to call the function in your tests